src/components/parameters/PropertyHistogramQuery-intervalField.yaml

```
@@ -1,6 +1,6 @@
 name: intervalField
 in: query
-description: Champ sur lequel sont définis les intervalles
+description: Champ sur lequel sont définis les intervalles.
```

:magic_wand: chore: Je pense qu'il ne faudrait pas mettre de point, les descriptions des autres paramètres n'en ont pas.
```suggestion
description: Champ sur lequel sont définis les intervalles
```